tip-wlan/charts/wlan-port-forwarding-gateway-service/templates/deployment.yaml

```diff
@@ -71,6 +71,10 @@ spec:
               value: {{ .Values.externallyVisible.host }}
             - name: PF_GATEWAY_RELEASE_EXT_PORT
               value: "{{ .Values.externallyVisible.port }}"
+            - name: EXT_PORT_RANGE_START
+              value: "{{ include "apDebugPortsStart" . }}"
+            - name: EXT_PORT_RANGE_END
+              value: "{{ include "apDebugPortsEnd" . }}"
```

Comment:
I think this should be `{{ sub (include "apDebugPortsEnd" . | atoi) 1 }}` to subtract 1 from the last value. See the value used in the now removed run.sh